src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const tbody = table.querySelector('tbody');
+const headers = table.querySelectorAll('th'); // Виправлено на querySelectorAll
+
+headers.forEach((th, index) => {
+  th.addEventListener('click', () => {
+    const rows = Array.from(tbody.querySelectorAll('tr'));
+    const isNumberColumn = index === 2 || index === 3;
```

Comment:
Consider adding a feature to toggle between ascending and descending order when a column header is clicked multiple times. This would enhance the user experience by allowing users to sort data in both directions.